isis/src/base/objs/CameraStatistics/CameraStatistics.truth

```diff
@@ -93,4 +93,4 @@ NorthAzimuth:
   NorthAzimuthMaximum = 360
   NorthAzimuthAverage = 356.025
   NorthAzimuthStandardDeviation = 28.4507
-
+  
```

Comment:
CameraStatistics unit test was failing due to the last line being deleted automatically by my text editor. I was unable to add a line back since my text editor automatically removes this line after adding and therefore are no changes to commit. Tried added back the line by editing the file within Github but it seems to still be failing due to it automatically adding a space. I can't seem to fix this. Any options for removing these changes altogether on this pr?